### PR TITLE
feat: RDCC-2278 Updated code to align with the Excel File structures

### PR DESCRIPTION
**Pull request by SujeethKumarAri**
### JIRA link  ###
https://tools.hmcts.net/jira/browse/RDCC-2278

### Change description ###
Updated code to align with the Excel File structures

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefFunctionalTest.java [hunk 1]
```diff
@@ -321,8 +321,8 @@ public void shouldThrowForbiddenExceptionForNonCompliantRole() {
     @ToggleEnable(mapKey = CASEWORKER_FILE_UPLOAD, withFeature = true)
     public void shouldUploadXlsxFileSuccessfully() throws IOException {
         ExtractableResponse<Response> uploadCaseWorkerFileResponse =
-                uploadCaseWorkerFile("src/functionalTest/resources/CaseWorkerUserWithNoPassword.xlsx",
-                        200, REQUEST_COMPLETED_SUCCESSFULLY,
+                uploadCaseWorkerFile("src/functionalTest/resources/CaseWorkerUserXlsxWithNoPassword.xlsx",
+                        200, CaseWorkerConstants.REQUEST_COMPLETED_SUCCESSFULLY,
```
Comment (AdilOoze) [inline src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefFunctionalTest.java:325]: Static import of `REQUEST_COMPLETED_SUCCESSFULLY` would be nicer

File: src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/service/ExcelUploaderTest.java [hunk 2]
```diff
@@ -95,7 +96,8 @@ public void parseXlsxShouldReturnWorkbookObjectTest() throws IOException {
                 getMultipartFile("src/integrationTest/resources/CaseWorkerUserXlsxWithNoPassword.xlsx",
                         TYPE_XLSX));
         List<CaseWorkerProfile> profiles = excelAdaptorService.parseExcel(workbook, CaseWorkerProfile.class);
-        assertThat(profiles).hasSize(workbook.getSheetAt(1).getPhysicalNumberOfRows() - 1);
+        assertThat(profiles).hasSize(workbook.getSheet(CaseWorkerConstants.REQUIRED_CW_SHEET_NAME)
```
Comment (AdilOoze) [inline src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/service/ExcelUploaderTest.java:99]: Static import of `REQUIRED_CW_SHEET_NAME` would be nicer

File: src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/CaseWorkerProfile.java [hunk 3]
```diff
@@ -30,15 +30,15 @@
...
 
-    @MappingField(columnName = "LAST NAME")
+    @MappingField(columnName = "Last Name")
     @NotEmpty
     private String lastName;
 
-    @MappingField(columnName = "Official Email", position = 1)
+    @MappingField(columnName = "Email", position = 1)
     @Email(message = CaseWorkerConstants.INVALID_EMAIL)
     @Pattern(regexp = CaseWorkerConstants.USER_NAME_PATTERN + "@"
```
Comment (AdilOoze) [inline src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/CaseWorkerProfile.java:43]: Static import of the constants would be nicer

File: src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefFunctionalTest.java (hunk 1 above)
Comment (SujeethKumarAri) [reply src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefFunctionalTest.java:325]: sure, will change :)

File: src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/service/ExcelUploaderTest.java (hunk 2 above)
Comment (SujeethKumarAri) [reply src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/service/ExcelUploaderTest.java:99]: sure, will change :)

File: src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/CaseWorkerProfile.java (hunk 3 above)
Comment (SujeethKumarAri) [reply src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/CaseWorkerProfile.java:43]: sure, will change :)

**Review by nitishsingla1234 — APPROVED** [review]

**Review by AdilOoze — APPROVED** [review]

**Review by nitishsingla1234 — APPROVED** [review]

**Pull request merged**